### PR TITLE
Added cobenefits_effect to mitigation actions

### PR DESCRIPTION
**Pull request by santostiago**
Updated the CSV file in AWS with the new version and included `quantified_effects` in the model and the importer.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -114,14 +114,43 @@
...
-    t.boolean "bur1"
+    t.boolean "quantified_effect"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["actor"], name: "index_mitigation_actions_on_actor"
     t.index ["ghg"], name: "index_mitigation_actions_on_ghg"
     t.index ["status"], name: "index_mitigation_actions_on_status"
   end
 
+  create_table "mitigation_effects", force: :cascade do |t|
```
Comment (agnessa) [inline db/schema.rb]: I think the effects part is not merged yet, maybe makes sense to keep away from schema here

File: app/serializers/api/v1/mitigation/mitigation_action_serializer.rb [hunk 2]
```diff
@@ -3,14 +3,17 @@
...
+#  name                         :text
 #  objectives                   :text
+#  mitigation_type              :string
 #  status                       :string
 #  actor                        :string
 #  time_horizon                 :string
 #  ghg                          :string
 #  estimated_emission_reduction :string
+#  cobenefits                   :string
+#  quantified_effect            :boolean
```
Comment (agnessa) [inline app/serializers/api/v1/mitigation/mitigation_action_serializer.rb]: There seem to be more changes to the annotation than the migration file suggests, is that right?

Comment (santostiago) [reply app/serializers/api/v1/mitigation/mitigation_action_serializer.rb]: I guess I just didn't run `annotate` on the previous commits. These fields have been changed in a previous PR, but didn't get annotated.

File: db/schema.rb (hunk 1 above)
Comment (santostiago) [reply db/schema.rb]: Yes, I needed to regenerate the schema. I just did another commit with it.

**Pull request merged**